src/App.js

```diff
@@ -1,5 +1,44 @@
+import { OUTPUT_MESSAGE_WINNER } from './lib/constants.js';
+import { print } from './lib/utils.js';
+import Input from './Input.js';
+import Race from './Race.js';
+
 class App {
-  async run() {}
+  #carArray;
+
+  #tryCount;
+
+  #winnerCarArray;
+
+  constructor() {
+    this.#carArray = [];
+    this.#tryCount = [];
+    this.#winnerCarArray = [];
+  }
```

Comment:
```suggestion
  #carArray = [];

  #tryCount = [];

  #winnerCarArray = [];
```

이렇게 초기화해주면 `constructor`가 불필요할 수도 있을 것 같아요. 이렇게 `constructor`에서 초기화시켜주시는 이유가 궁금합니다!

Reply:
프로퍼티를 constructor에서 초기화해주는 것이 좋다고 생각했습니다. 

하지만 이미 아래에서 다른 메서드에서 덮어쓸 것이기에 불필요하다는 생각이 드네요. 감사합니다.